indigo_app/templates/indigo_api/work_layout.html

```diff
@@ -72,6 +72,8 @@ <h5 class="main-header-title">
   <span class="work-title">{% if work.pk %}{{ work.title }}{% else %}Create a new work{% endif %}</span> 
   {% if work.stub %}<span class="badge badge-info">stub</span>{% endif %}
   {% if work.repealed_date %} <span class="badge badge-info">repealed</span> {% endif %}
+  {% if not work.commenced %} <span class="badge badge-info">not commenced</span>
+    {% elif not work.commencement_date %} <span class="badge badge-info">commencement uncertain</span> {% endif %}
```

Comment:
Normally `elif` is indented to the same level as `if`, so this line is overindented. Then the `endif` would come on a new line at the same indentation level.